pkg/driver/command_driver.go

```diff
@@ -71,7 +72,26 @@ func (d *CommandDriver) exec(op *driver.Operation) error {
 	}
 	cmd.Env = pairs
 	cmd.Stdin = bytes.NewBuffer(data)
-	out, err := cmd.CombinedOutput()
-	fmt.Fprintln(op.Out, string(out))
-	return err
+
+	// Make stdout and stderr from driver available immediately
+
+	stdout, _ := cmd.StdoutPipe()
+	go func() {
+		io.Copy(op.Out, stdout)
```

Comment:
This is a matter of taste, but did you consider handling the error?

Reply:
@glyn as with the stderr case I didnt handle the error here intentionally as I dont think we should fail the command because of a failure to produce output